csw-location/src/main/scala/csw/services/location/internal/Registry.scala

```diff
@@ -9,48 +9,45 @@ import csw.services.location.models.{Connection, Location}
 import scala.concurrent.duration.DurationDouble
 
 /**
-  * A `Registry` that manages [[akka.cluster.ddata.Replicator.Update]] and [[akka.cluster.ddata.Replicator.Get]] messages
-  * for both cluster data types namely, [[akka.cluster.ddata.LWWRegister]] and [[akka.cluster.ddata.LWWMap]]
+  * Registry is used to create CRDTs and manage its update and get messages for replicator
   *
-  * @param Key        The `Key` against which `update` or `get` will be performed
-  * @param EmptyValue The initial value passed to [[akka.cluster.ddata.Replicator.Update]]
-  * @tparam K The type of `Key`
-  * @tparam V The type of `ReplicatedData`
+  * @param Key        The CRDT key
+  * @param EmptyValue The default value of CRDT key
+  * @tparam K The type of CRDT key
+  * @tparam V The type of ReplicatedData
   */
 class Registry[K <: Key[V], V <: ReplicatedData](val Key: K, val EmptyValue: V) {
   /**
-    * Creates [[akka.cluster.ddata.Replicator.Update]] with the `Key`, `EmptyValue` and function `f` which will update
-    * majority nodes in cluster
+    * Creates an update message for replicator with WriteConsistency as majority
```

Comment:
Instead of writeconsistency, I will say it ensure that the response goes out only after majority nodes are written to.